test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -32,20 +32,6 @@ const ignoredMocks = [
 
 const v3IgnoredMocks = [
   'consent-granular.json', // TODO: Not yet implemented OKTA-594847
-  /** The below are all erroring due to the asterisk next to labels, OKTA-566071 will resolve this **/
-  'enroll-profile-new-checkbox.json',
-  'enroll-profile-new-custom-labels.json',
-  'enroll-profile-new-with-hcaptcha.json',
-  'enroll-profile-new-with-recaptcha-v2.json',
-  'enroll-profile-new.json',
-  'enroll-profile-with-idps.json',
-  'enroll-profile-with-password-returns-error.json',
-  'enroll-profile-with-password-returns-multiple-errors.json',
-  'enroll-profile-with-password.json',
-  'enroll-profile.json',
-  'error-new-signup-email-exists.json',
-  'error-new-signup-email.json',
-  /** END of asterisk issue */
```

Comment:
These were ignored because of the asterisk next to the label (rendered by MUI). We now either do not display the asterisk or when we do we apply the `no-translate` class to it to prevent it from failing the english leak test.

Reply:
💪